### DISCUSSION TITLE
Collapse boards

### DISCUSSION THREAD

**Discussion by rbx-max** [community/community#15588] [Projects and Issues]
Be nice if one could collapse the columns (buckets) when viewing in kanban board in github so one does not have to scroll (and the settings preserve by user in terms of which buckets and collapsed/expanded

**Comment by interob** [comment]
Would be great if columns can be collapsed, e.g. with the label text vertically:
![image](https://user-images.githubusercontent.com/19994913/189776184-ee5843d1-7f77-4a6c-a725-706852490bff.png)

**Comment by Jay-o-Way** [comment]
#6906
